src/node-lib/curriculum-api-2023/queries/lessonDownloads/getNextLessonsInUnit.ts

```diff
@@ -0,0 +1,25 @@
+import { LessonListSchema } from "./lessonDownloads.schema";
+
+const getNextLessonsInUnit = (unit: LessonListSchema, lessonSlug: string) => {
+  if (!unit || !lessonSlug) {
+    return [];
+  }
+  const lessonInUnit = unit.find((lesson) => lesson.lessonSlug === lessonSlug);
+  const lessonPosition = lessonInUnit?.orderInUnit;
+  const nextLessons =
+    lessonPosition &&
+    unit
+      .filter((lesson) =>
+        lesson.orderInUnit ? lesson.orderInUnit > lessonPosition : [],
+      )
+      .splice(0, 3);
```

Comment:
are the lessons always going to be sorted by orderInUnit?

Reply:
In the old API we order in the GQL query in the new API it's ordered in the MV